cheatsheets/gleam-for-python-users.md

```diff
@@ -4,33 +4,84 @@ title: Gleam for Python users
 ---
 
 - [Comments](#comments)
+    - [Python](#python)
+    - [Gleam](#gleam)
```

Comment:
Could you remove these extra contents links that say "Python" and "Gleam" please? Thank you

Reply:
It's my markdown plug-in that generate the TOC 🤔, will remove them